test/e2e/images_test.go

```diff
@@ -80,13 +80,13 @@ var _ = Describe("Podman images", func() {
 		session = podmanTest.PodmanNoCache([]string{"images"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
-		session.LineInOutputContainsTag("docker.io/library/alpine", "latest")
-		session.LineInOutputContainsTag("docker.io/library/busybox", "glibc")
-		session.LineInOutputContainsTag("foo", "a")
-		session.LineInOutputContainsTag("foo", "b")
-		session.LineInOutputContainsTag("foo", "c")
-		session.LineInOutputContainsTag("bar", "a")
-		session.LineInOutputContainsTag("bar", "b")
+		Expect(session.LineInOutputContainsTag("quay.io/libpod/alpine", "latest")).To(BeTrue())
+		Expect(session.LineInOutputContainsTag("quay.io/libpod/busybox", "latest")).To(BeTrue())
+		Expect(session.LineInOutputContainsTag("localhost/foo", "a")).To(BeTrue())
+		Expect(session.LineInOutputContainsTag("localhost/foo", "b")).To(BeTrue())
+		Expect(session.LineInOutputContainsTag("localhost/foo", "c")).To(BeTrue())
+		Expect(session.LineInOutputContainsTag("localhost/bar", "a")).To(BeTrue())
+		Expect(session.LineInOutputContainsTag("localhost/bar", "b")).To(BeTrue())
```

Comment:
This was a complete NOP. Plus, `LineInOutputContainsTag()` was broken. See below.